packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -1010,6 +1010,12 @@ abstract class BoxScrollView extends ScrollView {
 /// example of how a caller might wire up basic item selection, see
 /// [ListTile.selected].
 ///
+/// {@tool dartpad}
+/// This example shows a custom implementation of ListTile selection in a ListView.
+/// Long Press any ListTile to enable selection mode.
+/// ** See code in examples/api/lib/widgets/scroll_view/listview_select.1.dart **
```

Comment:
Nit:
```suggestion
/// This example shows a custom implementation of [ListTile] selection in a [ListView] or [GridView].
/// Long press any ListTile to enable selection mode.

/// ** See code in examples/api/lib/widgets/scroll_view/listview_select.1.dart **
```

Reply:
Since your sample includes a GridView, can you add it the sample to those docs as well?

Reply:
done